public/locales/en/panel-nightsky.json

```
@@ -0,0 +1,249 @@
+    }
+  },
+  "stars": {
+    "visibility": {
+      "title": "Visibility",
+      "input": {
+        "show-stars": "Show Stars",
+        "show-during-day": {
+          "label": "Show During Day",
+          "tooltip": "Check this box to show the stars during daytime"
+        }
+      }
+    },
+    "labels": {
+      "title": "Labels",
+      "input": {
+        "show-labels": "Show Labels",
+        "show-alternate-labels": "Show Alternate Labels"
+      }
+    },
+    "appearance": {
+      "title": "Apperance",
+      "warning": {
+        "title": "Only Night Sky Profile",
+        "description": "The buttons below only work in the 'nightsky' profile. Use the profile editor to edit the actions and adjust the values for your dome."
+      },
+      "buttons": {
+        "default-settings": "Default Settings",
+        "point-like-stars": "More Point Like Stars"
+      }
+    }
+  },
+  "sun": {
+    "glare": {
+      "title": "Glare",
+      "buttons": {
+        "show-glare": "Sow Glare",
+        "hide-glare": "Hide Glare"
+      }
+    },
+    "size": {
+      "title": "Size",
+      "buttons": {
+        "default-size": "Default Angular Size",
+        "large-size": "Large Angular Size",
+        "extra-large-size": "Extra Large Angular Size"
+      },
+      "aria-labels": {
+        "increase-size": "Increase angular size",
+        "decrease-size": "Decrease angular size"
+      },
+      "error": "Could not find Sun Angular Size settings"
+    },
+    "trails": {
+      "title": "Trails",
+      "buttons": {
+        "add-trail": "Add Trail",
+        "add-trail-simulation-date": "Add Trail for Simulation Date",
+        "add-trail-today": "Add Trail for Today",
+        "hide-trails": "Hide All"
+      },
+      "choose-date": "Choose Date",
+      "added-trails": {
+        "title": "Added Sun Trails",
+        "no-trails": "No sun trails"
+      }
+    }
+  },
+  "time": {
+    "simulation-time": {
+      "title": "Simulation Time",
+      "buttons": {
+        "set-now": "Set to Now"
+      },
+      "local": "Local",
+      "timezone": "Timezone",
+      "utc": "UTC",
+      "error": "Date out of range"
+    },
+    "jumps": {
+      "title": "Jumps",
+      "buttons": {
+        "sidereal-day": {
+          "decrease": "Go back in time one sidereal day",
+          "increase": "Go forward in time on sidereal day",
+          "title": "Sidereal Day"
+        },
+        "sidereal-week": {
+          "decrease": "Go back in time one sidereal week",
+          "increase": "Go forward in time one sidereal week",
+          "title": "Sidereal Week"
+        },
+        "solar-day": {
+          "decrease": "Go back in time one solar day",
+          "increase": "Go forward in time one solar day",
+          "title": "Solar Day"
+        },
+        "solar-week": {
+          "decrease": "Go back in time one solar week",
+          "increase": "Go forward in time one solar week",
+          "title": "Solar Week"
+        }
```

```suggestion
        "sidereal-day": {
          "title": "Sidereal Day",
          "decrease": "Go back in time one sidereal day",
          "increase": "Go forward in time on sidereal day"
        },
        "sidereal-week": {
          "title": "Sidereal Week",
          "decrease": "Go back in time one sidereal week",
          "increase": "Go forward in time one sidereal week"
        },
        "solar-day": {
          "title": "Solar Day",
          "decrease": "Go back in time one solar day",
          "increase": "Go forward in time one solar day"
        },
        "solar-week": {
          "title": "Solar Week",
          "decrease": "Go back in time one solar week",
          "increase": "Go forward in time one solar week"
        }
```

For stylistic consistency since the title was mentioned first in the other instances